app/controllers/admin/content_controller.rb

```diff
@@ -37,6 +37,24 @@ def edit
     new_or_edit
   end
 
+  def merge
+    if current_user.admin?
```

Comment:
Should something happen if the user is not an admin? I know it might not show up on the view, but someone could still potentially hit the URL anyway.